app/furniture/marketplace/current_person.rb

```diff
@@ -0,0 +1,17 @@
+class Marketplace
+  module CurrentPerson
+    def current_person
+      super
+
+      if @current_person.is_a?(Person) || @current_person.is_a?(Guest)
+        return @marketplace_current_person = @current_person
```

Comment:
I this this instance variable being set, `@marketplace_current_person`, but I don't ever see it referenced in this PR. What is it used for?

Reply:
Nope, that's some leftover that should probably be removed.